.github/workflows/build-plugins.yml

```diff
@@ -82,8 +82,17 @@ jobs:
         mkdir -p /tmp/zips/
         for plugin in /tmp/output/*; do
           zipname=/tmp/zips/$(basename ${plugin}).zip
-          cd $plugin
-          zip -r $zipname * -x "*assets*" -x "*.pnpm-store*" -x "tsconfig.json" -x "rollup.config.js" -x "*.yaml" -x "*.yml" -x "*.vscode*" -x "*.git*" -x "*.jpg" -x "*.png" -x "*.bmp"
+          # Names of the optional files (the license can either be called license or license.md, not both)
+          # (head is there to take the first file, because we're assuming there's only a single license file)
+          license="$(find $plugin -maxdepth 1 -type f \( -iname "license" -o -iname "license.md" \) -printf '%P\n' | head -n 1)"
+          readme="$(find $plugin -maxdepth 1 -type f -iname 'readme.md' -printf '%P\n')"
+          mainpy="$(find $plugin -maxdepth 1 -type f -name 'main.py' -printf '%P\n')"
+          # Add required plugin files (and directory) to zip file
+          zip -r $zipname "$plugin/dist" "$plugin/plugin.json" "$plugin/package.json"
+          # Check if other files exist, and if they do, add them
+          [ ! -z "$license" ] && zip -r $zipname "$plugin/$license"
+          [ ! -z "$readme" ] && zip -r $zipname "$plugin/$readme"
+          [ ! -z "$mainpy" ] && zip -r $zipname "$plugin/$mainpy"
         done
```

Comment:
An irregular exit occurs before this step can be finished.

Reply:
This error has been resolved as of the latest commit. It could have been fixed in two ways: wrap the optional file zips in parenthesis and append ` || true`, or, the other solution, which I decided to use, was to make use of proper if cases.